activerecord/lib/active_record/connection_handling.rb

```diff
@@ -180,12 +180,14 @@ def connected_to(database: nil, role: nil, shard: nil, prevent_writes: false, &b
     #
     # Usage:
     #
-    #   ActiveRecord::Base.connected_to(AnimalsRecord, MealsRecord], role: :reading) do
+    #   ActiveRecord::Base.connected_to([AnimalsRecord, MealsRecord], role: :reading) do
```

Comment:
Should we recommend using the splat in the documentation? The method name also needs a fix here:

```suggestion
    #   ActiveRecord::Base.connected_to_many(AnimalsRecord, MealsRecord, role: :reading) do
```